src/Associativity.h

```diff
@@ -83,13 +83,12 @@ struct AssociativeOp {
     AssociativePattern pattern;
     std::vector<Replacement> xs;
     std::vector<Replacement> ys;
-    bool is_associative;
+    bool is_associative{false};
 
-    AssociativeOp()
-        : is_associative(false) {
+    AssociativeOp() {
```

Comment:
= default. Maybe this change should go in after the default constructors one.

Reply:
All those changes are in https://github.com/halide/Halide/pull/5380

Reply:
Oh I see, you're merging the PRs into each other. Erm, OK I guess I'll just try to skim over the constructors.